proxy-node-shared/src/main/java/uk/gov/ida/notification/SamlFormViewBuilder.java

```diff
@@ -13,13 +13,17 @@
     public SamlFormView buildRequest(String url, AuthnRequest authnRequest, String submitText, String relayState) {
         String samlMessage = marshaller.transformToString(authnRequest);
         String encodedSamlMessage = Base64.encodeAsString(samlMessage);
+        return buildRequest(url, encodedSamlMessage, submitText, relayState);
+    }
+
+    public SamlFormView buildRequest(String url, String encodedSamlMessage, String submitText, String relayState) {
```

Comment:
Might be worth adding a unit test to cover this new signature.

Reply:
Yes, added unit tests for testing these public methods directly